help/en/releasenotes/current-draft-note.shtml

```diff
@@ -340,6 +340,7 @@
                 (3-rail).<br>
                 Use the LNCV Tool for programming.</li>
                 <li>Added Funk-Master LN+DCC 66410 definition. Renamed  model 66400 FunkMaster to Funk-Master LN.</li>
+                <li>Added a decoder definition to configure the 63450 LocoNet Display module, also with LNCV</li>
```

Comment:
This comment at on the wrong place. It's outside the `<ul>` tag.

Reply:
My apologize. I looked at the Merge commit and there it seemed wrong, but in the PR it's right. So it's correct.